cli/reportgen.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 cd tests/contract-playground
-forge build --ast
+forge build --ast --force
```

Comment:
Changing this forces forge to build AST without using cache files in `out` folder and this reduces the risk of MEP-related bugs  🐞  in generating report. (Helps in long term and healthy maintenance of this codebase)

However, we don't update the command within `aderyn` because we prioritize speed there.